### ISSUE TITLE
Issue with downloading a course

### ISSUE THREAD

**Issue by KoichaDev** [r00tmebaby/LinkedIn-Downloader#45]
So this is another issue from a course:

https://www.linkedin.com/learning/activate-connections-that-will-transform-your-career/connecting-at-work-creating-a-genuine-culture

The error I get is this:
https://imgur.com/E6yT8ZF

Could you please fix this? :)

**Comment by r00tmebaby** [comment]
https://www.lynda.com/Swift-tutorials/Learning-Swift-Playgrounds-Application-Development/2822374-2.html is the original course link. 

Use this program https://github.com/r00tmebaby/LinkedIn-Downloader/tree/master/LyndaDL%20Executable's and place the link into coursesList.txt

Here is a video to help you out  https://www.youtube.com/watch?v=FKPZeyM7098&t=6s

**Comment by KoichaDev** [comment]
Will try this out! Thanks! 

EDIT: Is there a way to get the whole playlist from the Lynda course instead of adding each URL link for each of the course section?

**Comment by r00tmebaby** [comment]
Yes, I have provided you with a script that dumps the whole course list and dumped list is also uploaded. 

Dump list script https://github.com/r00tmebaby/LinkedInDownloader/blob/master/Lynda%20Courses%20Dump/dumplist.py

Full Lynda course list (by 8th of March) https://github.com/r00tmebaby/LinkedIn-Downloader/blob/master/Lynda%20Courses%20Dump/LyndaCourseList%2008.03.2020.txt

**Comment by KoichaDev** [comment]
Fantastic! Thanks! 

Just want notify I am getting another error. I have attached the screenshot on the link below:

https://imgur.com/a/qzmD2ER

This is from this course:

https://www.linkedin.com/learning/lead-generation-foundations-2

**Comment by h4mp3l** [comment]
Thanks for this tool, it works  like a charm. Well mostly at least, i also got some problems when downloading subtitles. So some courses to not work with the LyndaDL executable.
I tried to get your scripts running through pycharm but when hitting start on the UI it threw the following error:
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":".artdeco"}
  (Session info: chrome=80.0.3987.132)
